backend/app/routes/teamMember/deleteTeamMember.js

```diff
@@ -1,3 +1,11 @@
+/* eslint-disable import/newline-after-import */
```

Comment:
[neat]please remove these un-necessary changes.

Reply:
@Kajol-Kumari @Kajol-Kumari If i remove those 4 line while commiting the changes it askes to change the entire file. Changing the entire file may effect other function so i added that comments. They are unnecessary. but to commit changes they are necessary other wise we have to change all those files. I hope you understand.

Reply:
the problem is these lines are disabling all the linting checks and so i can't allow it to be merged. can you please raise a new pr with just those 2 required change?